Cargo.toml

```diff
@@ -39,6 +39,7 @@ sl-compiler = { path = "compiler" }
 compiler_test_utils = { path = "compiler/test_utils" }
 slvm = { path = "vm" }
 shell = { path = "shell" }
+derive_more = {  version = "1.0.0", features = ["into", "from"]}
```

Comment:
I am not going to die on this hill and I know derive_more is pretty standard.  But do we really need a new dep for these "Loose" types?  Seems like it save a few lines of code and adds more derive macro compile time.  Your call though, just my two cents.